packages/tupaia-web/src/features/Map/Map.tsx

```diff
@@ -99,23 +102,25 @@ export const Map = () => {
     <MapContainer>
       <StyledMap bounds={entity?.bounds as LeafletMapProps['bounds']} shouldSnapToPosition>
         <TileLayer tileSetUrl={activeTileSet.url} showAttribution={false} />
-        <MapOverlays />
+        <PolygonLayer />
+        <MarkerLayer />
         <ZoomControl position="bottomright" />
-        <MapControlWrapper>
-          <MapControlColumn>
-            <MapOverlaySelector />
-            <MapLegend />
-          </MapControlColumn>
-          <TilePickerWrapper>
-            <TilePicker
-              tileSets={TILE_SETS}
-              activeTileSet={activeTileSet}
-              onChange={onTileSetChange}
-            />
-          </TilePickerWrapper>
-        </MapControlWrapper>
         <MapWatermark />
       </StyledMap>
+      {/* Map Controls need to be outside the map so that the mouse events on controls don't inter wit the map */}
+      <MapControlWrapper>
```

Comment:
restructure the controls to be outside the map to fix those scroll bugs that were happenning.